css/src/base/elements.css

```diff
@@ -5,7 +5,7 @@
 body {
   color: var(--color-fg);
   background: var(--color-bg);
-  font: normal 81.3%/1.538em var(--font-family);
+  font: normal 16px/1.5 var(--font-family);
```

Comment:
![screen shot 2018-11-26 at 17 29 32](https://user-images.githubusercontent.com/1845495/49008691-0a526200-f1a1-11e8-82f5-8fbb8e2f4a56.png)

It seems like that the line-height should be 1.2em according to the style guide. Is that correct @ckrina?

Reply:
<img width="999" alt="screen shot 2018-11-26 at 11 35 45" src="https://user-images.githubusercontent.com/9197058/49008972-859b2000-f16f-11e8-86ed-8ca2dc04449d.png">
@lauriii That text is false. Focus it and check the sidebar!

Reply:
I noticed that there's also mention of line heights of 24.5px so probably get confirmation from @ckrina what height we should be using 🙂

Reply:
Ok, I've been digging a little bit on it and finally got the point. The percentage used on Figma doesn't behave like the one in the browser, so the most reliable size are px there (24.5px). But I based the defined pixels for now (until we start using fluid type) based on that wrong %.
So I think it is better to stay with a rounded number (1.5) @zolhorvath defined which corresponds to the 24px. I'll update Figma.